Integrations/src/main/java/io/deephaven/integrations/python/PythonRunnable.java

```diff
@@ -0,0 +1,21 @@
+package io.deephaven.integrations.python;
+
+import org.jpy.PyObject;
+
+import java.util.Objects;
+
+/**
+ * A {@link Runnable} implementation which calls a Python callable.
+ */
+public class PythonRunnable implements Runnable {
+    private final PyObject pyCallable;
+
+    public PythonRunnable(PyObject pyCallable) {
+        this.pyCallable = Objects.requireNonNull(pyCallable);
```

Comment:
This is somewhat a stylistic and pragmatic approach IMO - to prefer the native JDK functions whenever possible; but also to avoid further entrenching the reliance on "Base" (which I feel has grown beyond what one might call "Base").

Reply:
I don't have any objections in this case.  If it was user-facing, I'd prefer them to know which param was bad.